### PR TITLE
Adding the Product Search tests.

### PR DESCRIPTION
**Pull request by nirupa-kumar**
@nnegrey Please review.
Apart from the tests, cleaned up the main files as well.
Currently missing the tests for ProductSearch.java file. Working on fixing some issues with the src and test files.

### CODE REVIEW

File: vision/product-search/cloud-client/src/test/java/com/example/vision/ImportProductSetsIT.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Integration (system) tests for {@link ImportProductSets}. */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class ImportProductSetsIT {
+  private static final String PROJECT_ID = "java-docs-samples-testing";
```
Comment (nnegrey) [inline vision/product-search/cloud-client/src/test/java/com/example/vision/ImportProductSetsIT.java]: Switch "java-docs-samples-testing" to `System.getenv("GOOGLE_CLOUD_PROJECT");`

File: vision/product-search/cloud-client/src/test/java/com/example/vision/ProductInProductSetManagementIT.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Integration (system) tests for {@link ProductInProductSetManagement}. */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class ProductInProductSetManagementIT {
+  private static final String PROJECT_ID = "java-docs-samples-testing";
```
Comment (nnegrey) [inline vision/product-search/cloud-client/src/test/java/com/example/vision/ProductInProductSetManagementIT.java]: `System.getenv("GOOGLE_CLOUD_PROJECT");`

File: vision/product-search/cloud-client/src/test/java/com/example/vision/ProductManagementIT.java [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Integration (system) tests for {@link ProductManagement}. */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class ProductManagementIT {
+  private static final String PROJECT_ID = "java-docs-samples-testing";
```
Comment (nnegrey) [inline vision/product-search/cloud-client/src/test/java/com/example/vision/ProductManagementIT.java]: `System.getenv("GOOGLE_CLOUD_PROJECT");`

File: vision/product-search/cloud-client/src/test/java/com/example/vision/ProductSetManagementIT.java [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Integration (system) tests for {@link ProductSetManagement}. */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class ProductSetManagementIT {
+  private static final String PROJECT_ID = "java-docs-samples-testing";
```
Comment (nnegrey) [inline vision/product-search/cloud-client/src/test/java/com/example/vision/ProductSetManagementIT.java]: `System.getenv("GOOGLE_CLOUD_PROJECT");`

File: vision/product-search/cloud-client/src/test/java/com/example/vision/ReferenceImageManagementIT.java [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** Integration (system) tests for {@link ReferenceImageManagement}. */
+@RunWith(JUnit4.class)
+@SuppressWarnings("checkstyle:abbreviationaswordinname")
+public class ReferenceImageManagementIT {
+  private static final String PROJECT_ID = "java-docs-samples-testing";
```
Comment (nnegrey) [inline vision/product-search/cloud-client/src/test/java/com/example/vision/ReferenceImageManagementIT.java]: `System.getenv("GOOGLE_CLOUD_PROJECT");`

**Comment by nnegrey** [comment]
@kurtisvg 
Hi Kurtis, question on Kokoro, it appears that the tests have passed successfully, but the success isn't getting passed back to the status check to merge. Any ideas?

**Comment by kurtisvg** [comment]
@nnegrey Looks like Kokoro was running tests on the previous commit. I retriggered it from this side and looks like it's happy now.

**Pull request merged**